torch/nn/modules/batchnorm.py

```diff
@@ -467,28 +467,31 @@ def forward(self, input):
 
     @classmethod
     def convert_sync_batchnorm(cls, module, process_group=None):
-        r"""Helper function to convert `torch.nn.BatchNormND` layer in the model to
-        `torch.nn.SyncBatchNorm` layer.
+        r"""Helper function to convert all :attr:`BatchNorm*D` layers in the model to
+        :class:`torch.nn.SyncBatchNorm` layers.
 
         Args:
-            module (nn.Module): containing module
+            module (nn.Module): module containing one or more attr:`BatchNorm*D` layers
             process_group (optional): process group to scope synchronization,
         default is the whole world
```

Comment:
I will add a followup to fix this indention

<img width="753" alt="Screen Shot 2020-06-08 at 10 31 50 AM" src="https://user-images.githubusercontent.com/16999635/84042610-4a3bfd00-a973-11ea-8f02-c1a7116c4af0.png">

Reply:
fixed!

Reply:
Thanks!